.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,18 +1,18 @@
 <!-- 
 Describe the changes you have made here: what, why, ... 
-Link issues that are fixed, e.g. "Fixes #333".
-If you fixed a koppor issue, link it, e.g. "Fixes https://github.com/koppor/jabref/issues/47".
-The title of the PR must not reference an issue, because GitHub does not support autolinking there.
+Link the issue that will be closed, e.g., "Closes https://github.com/JabRef/jabref/issues/333".
```

Comment:
I do prefer this version, but I don't know that it matters. 

```suggestion
Link the issue that will be closed, e.g., "Closes #333".
```

Reply:
It's actually the same, but the first one works better across repos with e.g. koppor#333

Reply:
o.O
I had not seen/paid attention to that version.